packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -455,16 +457,19 @@ const main = async () => {
         .argument('<path>', 'path or glob to format')
         .action(async (path) => {
             const prettier = p.join(require.resolve('prettier'), '..', '..', '.bin', 'prettier')
-            execSync(`${prettier} --write "${path}"`)
+            execSync(`"${prettier}" --write "${path}"`)
         })
 
     program
         .command('test')
+        .allowUnknownOption()
```

Comment:
I think this is useful as it allows you to pass in things like '--debug' to view the config file for jest. I'm open to reverting this if there is a good reason.